src/backend/api/reviews.js

```diff
@@ -0,0 +1,97 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../database");
+
+// GET - /api/reviews - returns all reviews
+
+router.get("/", async (req, res) => {
+  try {
+    const allReviews = await knex("review");
+    res.status(200).json(allReviews);
+  } catch (error) {
+    console.error("error");
+    res.status(500).send("Server Error");
+  }
+});
+
+// GET - /api/meals/:meal_id/reviews - returns all reviews for a specific meal
+// route for this endpoints is in meal.js file
+
+// POST - /api/reviews - adds a new review to the database
+
+router.post("/api/reviews", async (req, res) => {
```

Comment:
Since this is part of a nested router, can you see an issue with this path? What does it evaluate to if you account for how the reviews router is used in `app.js`? 🙂